### ISSUE TITLE
Fix dictionary `.clone()` to `.copy()` for gradients AutogradBatchNorm

### ISSUE THREAD

**Issue by Farbdrucker** [facebookresearch/CrypTen#20]
see my pull request
https://github.com/facebookresearch/CrypTen/pull/21

**Comment by lvdmaaten** [comment]
Good catch! There are some more issues with batchnorm in training mode -- will fix.
